fastlane/metadata/android/en-US/full_description.txt

```diff
@@ -32,4 +32,8 @@ Enjoy the freedom of the Matrix open standard! You have native interoperability
 Enjoy your right to private conversations - free from data mining, ads and all the rest of it - and stay secure. Only the people in your conversation can read your messages.
 
 <b>Chat across multiple devices</b>
-Stay in touch wherever you are with fully synchronized message history across all your devices, even those running Element legacy app, and on the web at https://app.element.io
\ No newline at end of file
+Stay in touch wherever you are with fully synchronized message history across all your devices, even those running Element legacy app, and on the web at https://app.element.io
+
+The application requires the android.permission.REQUEST_INSTALL_PACKAGES permission to enable the installation of applications received as attachments, ensuring seamless and convenient access to new software within the app.
+
+The application requires the USE_FULL_SCREEN_INTENT permission to ensure our users can effectively receive call notifications even when their devices are locked.
```

Comment:
Is this part required now?

Reply:
It's already live on the PlayStore and yes, it was required by Google. I forgot to add this part when importing the new description, and it's better to stay synced with what on the PlayStore. One day (probably when we will start translating this) publication will be automated as it was the case on EA.